doc/qos/SONiC_ROCEv2_support.md

```diff
@@ -397,93 +1260,157 @@ module: openconfig-qos
                  +--ro oc-qos-ext:queue?     string
                  +--ro oc-qos-ext:profile    -> ../../../../buffer-profiles/buffer-profile/name
 
+
+module: sonic-switch
+  +--rw sonic-switch
+     +--rw SWITCH
+        +--rw SWITCH_LIST* [switch]
+           +--rw switch               enumeration
+           +--rw fdb_aging_time?      uint32
+           +--rw rif_cntr_interval?   uint32
+           +--rw buffer_mode_lossless?  bool
+           +--rw roce_enable?   bool
 ```
 
 ### 3.6.2 CLI
 
-#### 3.6.2.1 Configuration Commands
+#### 3.6.2.1 CLICK Configuration Commands
+
+- Generate default Configurations for newly created ports  or ports does not have ROCE/buffer configurations and load it to config DB. This can be used to generate configs after DPB.
+
+```
+config qos reload --update --all-new-ports
+
+```
+- Clear only default ROCE qos maps, scheduler, wred etc.
+
+```
+config qos classifier clear --defaults
+```
+
+#### 3.6.2.2 Configuration Commands
+
 All commands are executed in `configuration-view`:
+
 ```
 sonic# configure terminal
 sonic(config)#
 ```
-##### 3.6.2.1.1 Initialize the buffer based on system defaults
-Initialize the buffer defaults based on platform specific values (ingress/ingress buffer pools size, buffer profile, priority-group, queue..etc)
+
+##### 3.6.2.2.1 Initialize the lossless buffers based on system defaults
+
+Initialize the lossless buffer defaults based on platform specific values (ingress/ingress buffer pools size, buffer profile, priority-group, queue..etc)
+
 ```
-sonic(config)# qos buffer init
+sonic(config)# qos buffer init lossless
 
 ```
 
-##### 3.6.2.1.2 Delete buffer pool
-Clear default buffer initialization.
+##### 3.6.2.2.2 Delete lossless buffers
+
+Clear buffer initialization.
+
 ```
 sonic(config)# no qos buffer init
 
 ```
-##### 3.6.2.1.3 Configure buffer pool
+
+##### 3.6.2.2.3 Configure buffer pool
+
 Configure shared head room size for the ingress buffer pool (fixed name - ingress_lossless_pool), other settings such as pool size and egress pool are automatically created during buffer init.
 Please refer user guide for platform specific defaults and based on that use these CLIs to fine-tune the numbers for the use-case.
+
 ```
 sonic(config)# qos buffer pool ingress_lossless_pool shared-headroom-size shared-headroom-size <xoff>

```

Comment:
How would user know the pool-name which has to be given in this command as we are having naming limitation in first release?

Reply:
yes, pool user can not create. he can use show running or show buffer pool cmd to know name

Reply:
thanks for the clarification